account_due_list/views/payment_view.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!--****************************** PAYMENTS ***********************************-->
+    <record id="view_payments_tree" model="ir.ui.view">
+        <field name="name">Payments</field>
+        <field name="model">account.move.line</field>
+        <field name="field_parent">partner_id</field>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <tree string="Payments" decoration-muted="reconciled!=False"
+                    decoration-danger="date_maturity&lt;current_date" create="false" delete="false">
```

Comment:
I would suggest to put here the following:
``` xml
decoration-danger="reconciled==False and date_maturity&lt;current_date"
```
This way, payments that are already reconciled are shown in grey rather than red